<launch>
	<node name="pose_estimation_node" pkg="pose_estimation_uibk" type="pose_estimation_node" output="screen" />
	<param name="path_to_config" value="config.txt" />
	<!-- path to object directory, change it based on your setup -->
	<param name="path_to_object_db" value="/home/pacman/CODE/pacman/poseEstimation/data/recognizedObjects" />
</launch>
                                                                                                                      

